### PR TITLE
removing register form

### PR DESCRIPTION
**Pull request by istumpf**
Screen Shot 2019-10-07 at 14 08 18
Screen Shot 2019-10-07 at 14 08 09

### CODE REVIEW

**Review by nolleto — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -843,22 +843,10 @@ <h2>Newsletter</h2>
         <div class="section-header">
```
Comment (paulodiovani) [inline index.html:844]: We must remove the whole section (or just comment out) and also from menu.

**Review by paulodiovani — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (istumpf) [reply index.html:844]: I think we should keep it until we have the photos available. Then we replace the whole section, with the event photos.

Comment (paulodiovani) [reply index.html:844]: Makes sense.
But have the `resgister` button and section with nothing bellow is weird.

For now the form says inscription is over, so won't be a problem to still show it.

**Comment by paulodiovani** [comment]
I think we should rather merge #19 instead of this,

**Comment by istumpf** [comment]
@paulodiovani  sounds good to me

**Pull request closed without merging**